### PR TITLE
UI improvements

### PR DESCRIPTION
**Pull request by NikosVogiatzis**
1. I centered the whole gui on main window and kept css on menu bar 
image

2. In freePlayWindow i removed the label shown when button is pressed and also resized the text on black keys so that it can contain the note name on one line. NOTE: i removed the number of octave because it was not necessaryto appear.
image

3.  In LearnWindow i also removed the label that showed the sequence of chord/scale but added the note display feature like above in free play. When a chord or scale button is pressed automatically the note name appears on the bottom of the button

image

### CODE REVIEW

File: src/main/java/sample/views/FreePlayWindow.java [hunk 1]
```diff
@@ -104,11 +119,40 @@ private void initialize() {
...
+        }
+
+        for(Button button : blackKeys)
+        {
+            String text = button.getTooltip().getText();
+            text = text.substring(0, text.length() -1);
+            button.setFont(new Font("aerials", 8));
+            button.setText(text);
+            button.setTextFill(Color.WHITE);
+            button.setAlignment(Pos.BOTTOM_LEFT);
```
Comment (Mjephter2) [inline src/main/java/sample/views/FreePlayWindow.java:146]: I would prefer to have this text centered on the black keys. Will create a separate issue for that.

File: src/main/java/sample/views/LearnWindow.java [hunk 2]
```diff
@@ -51,6 +54,13 @@ public class LearnWindow extends Application {
     private final GridPane keyPane = new GridPane();
     private final Button resetButton = new Button("RESET");
 
+    private final Button homeButton = new Button("Home");
+
+    private Text text = new Text("");
+
+    private ArrayList <Button> button_sequence = new ArrayList<>();
+
+    private String text_diplay_when_clicked = "";
```
Comment (Mjephter2) [inline src/main/java/sample/views/LearnWindow.java:63]: This variable is not really necessary. The places where it is used can make use can generate this text inline.

**Comment by NikosVogiatzis** [comment]
I also noticed that but with the command  
button.setAllignment(Pos.BOTTOM_CENTER); it still didnt center the  
text well. If u check by the end
of the keypane the black keys are not in line as they are in the  
beginning of key pane. I did not find why this happens, but if u press  
any black button(key)
then it gets automatically centered, which is not happening in white  
keys at all.
Παραθέτοντας από Jephter Maurice ***@***.***>:

**Comment by Mjephter2** [comment]
I'll merge in the changes. I'll address the observations in later issues. Thanks for your contributions. 🎉🍾

**Pull request merged**